projects/tsq-gallery/src/lib/components/tsq-gallery-viewer/tsq-gallery-viewer.component.ts

```diff
@@ -60,192 +53,152 @@ export class TSqGalleryViewerComponent {
   touchTimeStamp = 0;
   touchX = 0;
   initialTouchX = 0;
+  imageRotation = 0;
+  imageZoom = 1;
 
+  canGoForward = false;
```

Comment:
There are a lot of flag variables related to what you can do inside the GalleryViewer. I think would be better to wrap them in a class (something like GalleryViewerActions) to make everything more easier to read.

Reply:
I was thinking on doing something like that, but I came to the conclusion that I would end up just moving literally everything to that class which would result in the same problem there

Reply:
I will isolate the image positioning behaviors into a model file